lib/confluent_schema/server.ex

```diff
@@ -13,6 +13,12 @@ defmodule ConfluentSchema.Server do
     GenServer.start_link(__MODULE__, opts, name: name)
   end
 
+  @doc "Used on test setup to block test until server has started."
+  def wait_start(), do: GenServer.call(__MODULE__, :wait_start)
```

Comment:
You could drop this and instead do `:sys.get_state` in the test code (which you could wrap behind some test-only helper like `sync` or `wait_start`).